### ISSUE TITLE
[BUG] No default font size on web pages

### ISSUE THREAD

**Issue by hdhalter** [opensearch-project/project-website#999]
**Describe the bug**
The main pages of the website don't have a default font setting, so if you have modified font settings on your browser, the font sizes across the website will be different. This causes a bad experience because it should be consistent as you move across the site. 

**To Reproduce**
1. Go to your browser settings and change the font size and/or zoom level. 
![image](https://user-images.githubusercontent.com/104463754/190281799-07e4ea28-e5d4-4130-994f-e34d61e46134.png)

2. Go to https://opensearch.org/about.html.
3. Open another window or tab and go to https://opensearch.org/docs/latest.
4. Notice the size difference. Switch between the windows to see that it's very noticeable.

**Expected behavior**
The website pages should have a default font size assigned like the documentation pages. So, if the browser font settings are modified, the entire site, including the documentation page will ignore the updated settings and we'll have consistency. 

**Plugins**
n/a

**Screenshots**

**Host/Environment (please complete the following information):**
 - OS: [e.g. iOS] Mac
 - Version: [e.g. 22] OpenSearch 2.3
 - Browsers: [e.g. Chrome, Safari] Chrome was fine; Firefox was the problem
 - Browser Version: [e.g 93.0.4521.12]

**Additional context**
I tested this with Miki and he confirmed that there was only a default font setting on the documentation page.

**Comment by nateynateynate** [comment]
Can confirm. Computed styles put one page at 1.8em for font size and the other at 1. 

I'll trace the CSS back up to see where that's getting overridden and see if I can fix that for you. 

Thanks for pointing it out @hdhalter !

**Comment by nateynateynate** [comment]
I have filed #1046 which might be related - at some point we overrode our Open Sans font and font-size defaults and no one really noticed.

**Comment by hdhalter** [comment]
No, it's not fixed. Changing the font size on the browser (Firefox) affects all the website pages except Documentation pages.
